examples/with-amp/README.md

```diff
@@ -1,26 +1,30 @@
-[![Deploy to now](https://deploy.now.sh/static/button.svg)](https://deploy.now.sh/?repo=https://github.com/zeit/next.js/tree/master/examples/with-amp)
+[![Deploy to now](https://deploy.now.sh/static/button.svg)](https://deploy.now.sh/?repo=https://github.com/zeit/next.js/tree/master/examples/experimental-amp)
```

Comment:
This readme is inaccurate. Can we switch back to the old one with some updates?

Reply:
Yeah, updated